samples/sample_multi_transcode/src/sample_multi_transcode.cpp

```diff
@@ -358,90 +359,98 @@ void Launcher::DoTranscoding()
 {
     mfxStatus sts = MFX_ERR_NONE;
 
-    // get parallel sessions parameters
-    mfxU32 totalSessions = (mfxU32)m_pSessionArray.size();
-    MSDKThread* pthread = 0;
-
-    for (mfxU32 i = 0; i < totalSessions; i++)
+    // Run transcoding threads
```

Comment:
this one is obvious comment, we don't need it. function name is telling whats happening

Reply:
Fixed